Classes7.3/IPrint.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Classes7._3
+{
+    public interface IPrint
+    {
+        public string Description { get;}
```

Comment:
You can remove from here and make this property virtual in abstract class

Reply:
Got it, reorganized this project a bit. Fixed.